libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -345,6 +345,8 @@ class BaseChatOpenAI(BaseChatModel):
     http_async_client: Union[Any, None] = None
     """Optional httpx.AsyncClient. Only used for async invocations. Must specify 
         http_client as well if you'd like a custom client for sync invocations."""
+    stop: Optional[Union[List[str], str]] = Field(default=None, alias="stop_sequences")
```

Comment:
If I initialize ChatOpenAI with `stop` now, I get a warning:
```
UserWarning: WARNING! stop is not default parameter.
stop was transferred to model_kwargs.
Please confirm that stop is what you intended.
```
Are we un-doing a change here?

Reply:
i think this should have opposite effect?